pages/docs/guides/validate.md

```diff
@@ -0,0 +1,125 @@
+
+Errors in your document are highlighted with a red underline, showing which lines are invalid. The console also provides feedback, allowing you to further troubleshoot with detailed error messages. When a document is invalid, it provides the following error: `Empty or invalid document please fix errors / define AsyncAPI document`.
+
+#### AsyncAPI CLI validation 
+The following [AsyncAPI CLI](https://github.com/asyncapi/cli#installation) command validates AsyncAPI documents in your local computer or in CI/CD automation:
+
+ ```
+ asyncapi validate asyncapi.yaml
+
+ ```
+
+<Remember>
+
+You can also open AsyncAPI Studio from the CLI by running the command `asyncapi start studio`.
+
+</Remember>
+
+#### Parsers (code) validation 
+AsyncAPI provides official [JavaScript](https://github.com/asyncapi/parser-js) and [Go](https://github.com/asyncapi/parser-go) parsers for validating AsyncAPI documents. 
+
+<Remember>
+Official parsers use <a href='https://github.com/asyncapi/spec-json-schemas/'>JSON Schema</a> created for AsyncAPI specification. JSON Schema is not enough to fully validate AsyncAPI documents. Learn more about <a href='https://github.com/asyncapi/spec-json-schemas#custom-validation-needs'>custom JSON Schemas validation needs</a>. Official JavaScript parser supports and validates these special needs.
+
+Take it into account if you're thinking about writing your own parser using official JSON Schema.
+</Remember>
+
+### Validation against best practices or company governance rules
+Now let's discuss options for validating against best practices or company governance rules, also known as **linting**. When various teams use AsyncAPI, you want to ensure they follow the same rules and are consistent across the organization. It is not enough to validate AsyncAPI documents against official specification rules. 
+
+<Remember>
+
+Let's discuss an example. While the `summary` property is optional in an AsyncAPI document, you could choose to require it for your organization. You would then implement a solution that enables you to enforce internal rules on AsyncAPI documents' providers.
+
+</Remember>
+
+One way this can be done is by using **Spectral**, an API linting tool which has a built-in [custom ruleset properties](https://meta.stoplight.io/docs/spectral/e5b9616d6d50c-custom-rulesets) with [AsyncAPI rules](https://meta.stoplight.io/docs/spectral/1e63ffd0220f3-async-api-rules) for the AsyncAPI specification. It also enables you to define company-specific rules that you can use internally.  
+
+To get started:
+1. Install [Spectral](https://meta.stoplight.io/docs/spectral/b8391e051b7d8-installation). 
+2. Create a file named `.spectral.yaml` to begin writing your API description and document rules. 
```

Comment:
please play a bit with this numbered list in some editor, it is breaking at the moment when rendered from markdown to HTML

<img width="2025" alt="Screenshot 2022-11-15 at 18 29 43" src="https://user-images.githubusercontent.com/6995927/201986852-1d1121a9-4558-48cd-90e7-a47aa8984d76.png">

you see, indents are wrong

for example step 2 should look like 👇🏼 to have proper indent
```
2. Create a file named `.spectral.yaml` to begin writing your API description and document rules. 
    Example:
    ```js
    {
      "formats": ["asyncapi2"],
      "extends": "spectral:asyncapi",
      "rules": {
        // add your own rules here
      }
    }
    ```
```